src/api/app/views/webui2/webui/staging/workflows/_overall_state.html.haml

```diff
@@ -1,7 +1,7 @@
 %span.badge{ class: "state-#{staging_project.overall_state}" }
   %span.badge.badge-light
     -# TODO: Link to the staging project show page
-    = link_to(staging_project.staging_identifier, project_show_path(staging_project.staging_identifier))
+    = link_to(staging_project.staging_identifier, project_show_path(staging_project))
```

Comment:
Why not keep it like before? If the name doesn't fit the naming pattern, then it's just displayed. Otherwise, it's only the staging identifier as with the suggested pattern

Reply:
You mean to move the change I made from the staging_identifier method to this link, and wherever else we need it? :thinking:

I think we can move this method to a helper and rename it. Essentially we want something that gives us the letter if it fits the default naming scheme and otherwise return the full name.

Reply:
I mean that the change to staging_identifier implies that you don't need to change it in other places, like this line.